crates/rlp/src/lib.rs

```diff
@@ -41,3 +49,17 @@ pub const EMPTY_STRING_CODE: u8 = 0x80;
 
 /// RLP prefix byte for a 0-length array.
 pub const EMPTY_LIST_CODE: u8 = 0xC0;
+
+// Not public API.
+#[doc(hidden)]
+#[deprecated(since = "0.3.0", note = "use `Error` instead")]
```

Comment:
is this tag functional with `doc(hidden)` ?

Reply:
Do you mean `deprecated`? If so, yes, it worked on the other crates in this repo

Reply:
okay great 👌